server/src/core/auth/auth.module.ts

```diff
@@ -1,32 +1,30 @@
 import { Module } from '@nestjs/common';
 import { JwtModule } from '@nestjs/jwt';
-import { ConfigModule, ConfigService } from '@nestjs/config';
 import { JwtAuthStrategy } from './strategies/jwt.auth.strategy';
 import { AuthService } from './services/auth.service';
 import { GoogleAuthController } from './controllers/google-auth.controller';
 import { GoogleStrategy } from './strategies/google.auth.strategy';
 import { PrismaService } from 'src/database/prisma.service';
 import { UserModule } from '../user/user.module';
 import { VerifyAuthController } from './controllers/verify-auth.controller';
-
 import { TokenService } from './services/token.service';
 import { AuthResolver } from './auth.resolver';
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
```

Comment:
I am not sure about EnvironmentService. It looks like ConfigService was more generic (and could have stuff that's not setted in the environment variables themselves). I understand that you want to have a safer interface than configService.get('xxx') which is not very reliable. However not sure about the naming maybe rename EnvironmentService to ConfigService?

Reply:
Read the EnvironmentService code. It provides value as it is.
I feel we should still have a ConfigService on top of EnvironmentService, which is actually similar to your Settings.
Something like: anywhere in the app ==> SettingsService ==> EnvironmentService